deployments/container/Dockerfile

```diff
@@ -46,7 +50,16 @@ ENV GOPATH=/go
 ENV PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
 
 WORKDIR /build
-COPY . .
+
+# Copy everything that is needed for the go build, but do not invalidate go
+# build artifact layer unnecessarily often. Note that --exclude only affects the
+# cache.
+COPY    --exclude=./templates/** \
+        --exclude=./deployments/helm/** \
+        --exclude=./hack/kubelet-plugin-prestart.sh \
+        --exclude=./demo/** \
+        --exclude=./Dockerfile \
+        . .
```

Comment:
did that for faster iteration during local dev, will move this to a separate patch